gruntfileConfig.json

```diff
@@ -4,6 +4,7 @@
   "src": "./src",
   "assets": "./src/assets",
   "theme": "./node_modules/patternpack-example-theme",
+  "logo": "/theme-assets/images/pp-logo.svg",
```

Comment:
One other thing.  The default values should be placed in BOTH `gruntfileConfig.json` and in the defaults section of `tasks/patternpack.js`